### DISCUSSION TITLE
241126_화

### DISCUSSION THREAD

**Discussion by Soy17** [boostcampaitech7/level2-cv-semanticsegmentation-cv-19-lv3#59] [TIL]
- [ ] 2048 b0-b4 진행해보기 
`mixed precision, batch size 2로 해보기💌`

**Comment by wonjeongjeong** [comment]
- [x] stratified group kfold 마무리
- [ ] 퀴즈 8~10

**Comment by june21a** [comment]
- [x] upernet + hrnet18 validation mDice 0.99까지 나오는 것 마무리하고 제출
- [x] elastic transform 앙상블 실험
- [x] elastic transform 앙상블 실험 이후 좋았던 augmentation 모두 합쳐서 실험해보기
- [x] 수면

**Comment by minrongtic** [comment]
**TO DO** 

- [ ] 각도 eda (되나?) -> crop rotate augmentation

**Comment by Yoon0717** [comment]
- [x] gradient accumulation 사용해서 2048 학습

**Comment by sejongmin** [comment]
- [x] resume 코드 작성해보기
- [x] 라벨스무딩 도전해보기
